<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="StackWinSyst">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="WinLogin">
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>12</number>
       </property>
       <item row="1" column="1" colspan="3">
        <widget class="QLabel" name="TextPassword">
         <property name="text">
          <string>请输入密码：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="EditPassword">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="BtnSystLoginExit">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="BtnSystLoginOK">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="BtnSystLoginPassword">
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>修改密码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" rowspan="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4" rowspan="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="WinPassword">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0,1" columnstretch="1,0,0,1">
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="EditPwdNew">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <spacer name="SpacerT">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLabel" name="TextPwdNewR">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>请再次输入新密码：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="EditPwdOld">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="EditPwdNewR">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="TextPwdNew">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>请输入新密码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="TextPwdOld">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>请输入原始密码：</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <spacer name="SpacerB">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="BtnSystPasswordExit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="BtnSystPasswordOK">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>97</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>确认修改</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="7">
        <spacer name="SpacerL">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3" rowspan="7">
        <spacer name="SpacerR">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="WinSyst">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,0,0">
       <item>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,1">
         <item row="0" column="0">
          <widget class="QLabel" name="TextUser">
           <property name="text">
            <string>操作用户：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="BoxUser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>操作员</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>管理员</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2" rowspan="3">
          <widget class="QTextBrowser" name="TextDebug"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextMode">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>启动方式：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="BoxMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>按钮启动</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>脚踏启动</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>滑罩启动</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>网络启动</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="TextStyle">
           <property name="text">
            <string>样式风格:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="BoxStyle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>97</width>
             <height>44</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>孔雀绿</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>濡雨蓝</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>网络数据库配置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>sa</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>123</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>192.168.1.158</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>Task Data</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>密码:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>服务器IP地址:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>数据库名称:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>帐号:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>本地IP地址:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="EditIp">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>网关:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>44</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="TabDisk">
         <column>
          <property name="text">
           <string>文件系统</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>容量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>已用</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>可用</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>已用%</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>挂载点</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_13" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BtnExit">
            <property name="minimumSize">
             <size>
              <width>97</width>
              <height>44</height>
             </size>
            </property>
            <property name="text">
             <string>返回</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
